ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/fastsync/FastSyncActions.java

```diff
@@ -80,14 +80,8 @@ public FastSyncActions(
       return waitForAnyPeer().thenApply(ignore -> fastSyncState);
     }
 
-    final WaitForPeersTask waitForPeersTask =
-        WaitForPeersTask.create(
-            ethContext, syncConfig.getFastSyncMinimumPeerCount(), metricsSystem);
-
     LOG.debug("Waiting for at least {} peers.", syncConfig.getFastSyncMinimumPeerCount());
-    return ethContext
-        .getScheduler()
-        .scheduleServiceTask(waitForPeersTask)
```

Comment:
@ajsutton - do you know if there was a reason we were scheduling the `waitForPeersTask` asynchronously?  I cut this part, which should be fine as far as I can tell.

Reply:
I'd like to say there was some sequence of refactoring that led to this and a final step just got missed, but most likely I was just an idiot when I wrote it initially. :)

Reply:
🤣